### PR TITLE
Added web interface for data.json input of challenges and apis

### PR DESCRIPTION
**Pull request by theCreedo**

### CODE REVIEW

File: app/admin_app.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    if request.method == 'GET':
+		return render_template('form.html')
+    else:
+        val = request.form.get('button')
+        if val == 'apis':
+            return apis(request)
+        elif val == 'challenges':
+            return challenges(request)
+        else:
+			return render_template('form.html')
```
Comment (rjchee) [inline app/admin_app.py]: why is this indentation so weird

File: app/data.json [hunk 2]
```diff
@@ -1,40 +1,41 @@
...
-        "rest",
-        "restful",
-        "jquery",
-        "capital",
-        "one"
-      ],
       "chip_keywords": [
-        "REST",
-        "banking",
+        "REST",
```
Comment (rjchee) [inline app/data.json]: are these extra spaces

File: app/admin_app.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+def apis(request):
+	api_info = {
+		'name':request.form.get('company_name'),
+		'image': request.form.get('image'),
+		'description': request.form.get('description'),
+		'link': request.form.get('link'),
+		'keywords': request.form.get('search_keywords').split(','),
```
Comment (rjchee) [inline app/admin_app.py]: `[keyword.strip() for keyword in request.form.get('search_keywords').split(',')]` so we can do split on comma-space as well as comma

File: app/admin_app.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+	update_api_json(api_info)
+	return render_template('form.html')
+
+
+def challenges(request):
+	challenge_info = {
+		'name':request.form.get('company_name'),
+		'image': request.form.get('image'),
+		'link': request.form.get('link'),
+		'keywords': request.form.get('search_keywords').split(','),
```
Comment (rjchee) [inline app/admin_app.py]: same here and for the chip keywords

**Review by rjchee — CHANGES_REQUESTED** [review]
almost good

File: app/admin_app.py (hunk 1 above)
Comment (theCreedo) [reply app/admin_app.py]: I don't see that in my code... rip. I'll reindent

File: app/admin_app.py (hunk 3 above)
Comment (theCreedo) [reply app/admin_app.py]: Will do

**Pull request merged**